CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.22.2
+
+**Release date:** 2022-08-29
+
+This release fixes a bug related scaling up the canary deployment when a
+reference to an autoscaler is specified.
+
+Furthermore, it contains updates to packages used by the project, including
+updates to Helm and grpc-health-probe used in the loadtester.
+
+CVEs fixed:
```

Comment:
Is it worth mentioning that all of these CVEs were in Flagger dependencies and not in Flagger itself?